### PR TITLE
Preserve the font size as one navigates through the CIDER inspector

### LINKED ISSUE #3527: Inspector: keep the font size
I noticed that while using the inspector, if I `M-x text-scale-increase`, that font size won't be kept as one navigates.

### PR DESCRIPTION
**Pull request by vemv**
Fixes #3527

### CODE REVIEW

File: cider-inspector.el [hunk 1]
```diff
@@ -388,8 +388,10 @@ MAX-COLL-SIZE if non nil."
 ;; Render Inspector from Structured Values
 (defun cider-inspector--render-value (value)
   "Render VALUE."
-  (cider-make-popup-buffer cider-inspector-buffer 'cider-inspector-mode 'ancillary)
-  (cider-inspector-render cider-inspector-buffer value)
+  (let ((font-size (when-let ((b (get-buffer cider-inspector-buffer)))
```
Comment (bbatsov) [inline cider-inspector.el:391]: I'd add a note here explaining why we need to do this.

**Comment by bbatsov** [comment]
I'm guessing we might have a similar problem in other buffers with drill-down functionality.

**Comment by vemv** [comment]
> I'm guessing we might have a similar problem in other buffers with drill-down functionality.

I could use a refresher as for which are those!

**Comment by bbatsov** [comment]
I could use one myself. 😆 I just have some vague memory there was something else besides this. (maybe the ns browser?) Anyways, not a big deal.

**Pull request merged**